src/packages/builder/src/codegen/index.ts

```diff
@@ -14,13 +15,20 @@ const content = `/* eslint-disable */
 */`;
 
 export const codeGenerator = async (schema: string, options?: CodegenOptions) => {
+	const extensions = '/**/!(*.generated).{ts,tsx}';
+
 	try {
 		const files = await executeCodegen({
 			cwd: process.cwd(),
 			pluginLoader: async (plugin: string) => import(plugin),
 			schema,
 			ignoreNoDocuments: true,
-			documents: ['./src/**/!(*.generated).{ts,tsx}'],
+			documents: [
+				path.join('./src', extensions),
```

Comment:
Does this generate the types in both
`path.join('./src', extensions),`
and if provided in these paths?

```
watchForFileChangesInPaths?.map((clientPath) =>
					path.join(clientPath, extensions)
				) || []),
				```

Reply:
I have just tested if the same path appears in both and it works. :)